packages/elements/src/overlay/elements/overlay.ts

```diff
@@ -96,10 +96,10 @@ const shouldRefitProperties: string[] = ['position', 'x', 'y', 'positionTarget',
 
 /**
  * Element to help building modals, dialogs and other overlay content
- * @fires closed - Dispatched when overlay becomes invisible and close animation finishes
- * @fires opened - Dispatched when the overlay becomes visible and the open animation finishes
- * @fires refit - Dispatched when refit algorithm finishes calculations
- * @fires opened-changed - Dispatched when when open attribute changes internally. Prevent default to stop opening/closing pipeline
+ * @fires closed - Fired when overlay becomes invisible and close animation finished
+ * @fires opened - Fired when overlay becomes visible and the open animation finished
+ * @fires refit - Fired when refit algorithm finishes calculations
+ * @fires opened-changed - Fired when the user changed open state of overlay e.g. when users press escape or use close button to close the overlay. The event is not triggered if `opened` property is changed by programmatically.
```

Comment:
Fired when the user change"s"~~d~~ open state of overlay e.g. when "the" user~~s~~ press"es" escape "key" or use"s" close button to close the overlay. The event is not triggered if `opened` property is changed ~~by~~ programmatically.